Library/ViewModels/DeprecatedWebViewModel.swift

```diff
@@ -41,7 +41,7 @@ DeprecatedWebViewModelOutputs {
       // Hide loading if the web view fails with a non-102 error code (102 is the interrupted error that
       // occurs anytime we cancel a request).
       self.webViewDidFailErrorProperty.signal
-        .filter { ($0 as? NSError)?.code != .some(102) }
+        .filter { ($0 as NSError?)?.code != .some(102) }
```

Comment:
don't know why this works, but whatever it does. feel there is some error error/nserror bridging going on, and not sure i care enough to know why...